File: algoritimos2.0/exercicio_primeira_aula/exercicio1.c
#include <stdio.h>

int main(void){
	int cont=0;
	char num[50];

	scanf("%s",&num);

	while(num[cont]!='\0'){
		cont++;
	}
	cont--;
	while(cont>=0){
		printf("%c",num[cont] );
		cont--;
	}

	return 0;

}

File: programacao_orientada/lista1/exercicio1.java
import java.util.Scanner;

public class exercicio1{
	public static void main(String[] args){
		Scanner tec = new Scanner(System.in);

		int num = tec.nextInt();
		int cont=1;

		while(cont<num){
			System.out.printf(cont+" ");
			cont=cont+2;
		}
	}

}